pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,86 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("create", "edit", "list", "update", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "list", "update", "watch"),
+					Resources: sets.NewString("projects"),
+				},
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("users", "groups"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("sar", "lar"),
+				},
+			},
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryViewerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "watch"),
+					Resources: sets.NewString("projects"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("sar", "lar"),
+				},
+			},
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryEditorRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("edit", "list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "watch"),
+					Resources: sets.NewString("projects"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("sar", "lar"),
+				},
+			},
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryBasicUserRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "list", "watch"),
```

Comment:
I would not expect a basic user to be able to create projects... to be effective, that would have to be a cluster-scoped operation, which would be super-powerful